ksqldb-engine/src/main/java/io/confluent/ksql/engine/RuntimeAssignor.java

```diff
@@ -34,7 +34,7 @@
 import org.slf4j.LoggerFactory;
 
 public class RuntimeAssignor {
-  private final Map<String, Collection<SourceName>> runtimesToSources;
+  private final Map<String, Collection<String>> runtimesToSources;
```

Comment:
Should this be renamed `runtimesToTopics` now, so it's clear for readers what this actually is? (And similarly for the other changes in this PR)